test/e2e/scaffold/scaffold.go

```diff
@@ -637,19 +634,6 @@ func waitExponentialBackoff(condFunc func() (bool, error)) error {
 	return wait.ExponentialBackoff(backoff, condFunc)
 }
 
-// generateWebhookCert generates signed certs of webhook and create the corresponding secret by running a script.
-func generateWebhookCert(ns string) error {
-	commandTemplate := `testdata/webhook-create-signed-cert.sh`
```

Comment:
It seems that the old script is not required anymore? We should delete it

Reply:
Done